cost-analyzer/templates/kubecost-cluster-controller-template.yaml

```diff
@@ -222,10 +221,11 @@ spec:
         {{- if .Values.clusterController.labels }}
         {{- toYaml .Values.clusterController.labels | nindent 8 }}
         {{- end }}
-      {{- with .Values.global.podAnnotations}}
       annotations:
+      {{- with .Values.global.podAnnotations}}
         {{- toYaml . | nindent 8 }}
       {{- end }}
+        checksum/configs: {{ include "configsChecksum" . }}
```

Comment:
Good catch!

Reply:
Thanks. See updated testing in notes above and added some details